autopilot_test.go

```diff
@@ -155,4 +49,25 @@ var _ = Describe("Command Syntax", func() {
 		args := cliConn.CliCommandArgsForCall(0)
 		Ω(args).Should(Equal([]string{"push", "-h"}))
 	})
+
+	It("can push an app that already exists", func() {
+		cliConn.CliCommandReturns([]string{"myapp and-other-stuff"}, nil)
+		output := CaptureOutput(func() {
+			autopilotPlugin.Run(cliConn, []string{"push-zdd", "myapp"})
+		})
+
+		Expect(len(ActionList)).To(Equal(3))
+		Expect(output).To(ContainElement(ContainSubstring("using zero-downtime-deployment")))
+	})
+
+	It("can push an app that doesn't exist", func() {
+		cliConn.CliCommandReturns([]string{"some-other-app and-other-stuff"}, nil)
+		output := CaptureOutput(func() {
+			autopilotPlugin.Run(cliConn, []string{"push-zdd", "my-new-app"})
+		})
+
+		Expect(len(ActionList)).To(Equal(1))
+		Expect(output).ToNot(ContainElement(ContainSubstring("using zero-downtime-deployment")))
```

Comment:
similar to above,
we dont need to test for a specific string being printed. CLI output isnt really our concern in this particular behavior we are testing. It also might change, which would fail our test.

consider using:
`CliCommandCallCount ` will get you the call count
https://github.com/cloudfoundry/cli/blob/master/plugin/fakes/fake_cli_connection.go#L267

`CliCommandArgsForCall` will get you the actual argument sets called.
https://github.com/cloudfoundry/cli/blob/master/plugin/fakes/fake_cli_connection.go#L273